plonk-core/src/proof_system/interface.rs

```diff
@@ -0,0 +1,475 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
```

Comment:
Is this file copied from Dusk?
Otherwise please fix the copyright.

Reply:
Fixed